.devcontainer/devcontainer.json

```
@@ -40,12 +40,12 @@
         "source=${localWorkspaceFolder}/data/calibrations/zed,target=/usr/local/zed/settings,type=bind,consistency=delegated",
         "type=bind,readonly,source=/etc/localtime,target=/etc/localtime"
     ],
-    "image": "ghcr.io/missourimrdt/autonomy-jammy:2025-05-08-03-09-25",
-    // "image": "ghcr.io/missourimrdt/autonomy-jetpack:2025-05-08-03-09-25",
-    // "build": {
-    //     "dockerfile": "Jammy.dockerfile"
-    //     "dockerfile": "JetPack.dockerfile"
-    // },
+    // "image": "ghcr.io/missourimrdt/autonomy-jammy:2025-05-10-00-39-31",
+    // "image": "ghcr.io/missourimrdt/autonomy-jetpack:2025-05-10-00-39-31",
+    "build": {
+        "dockerfile": "Jammy.dockerfile"
+        // "dockerfile": "JetPack.dockerfile"
+    },
```

Switch back to defaulting to prebuild Jammy Image before merge.
```suggestion
    "image": "ghcr.io/missourimrdt/autonomy-jammy:2025-05-10-00-39-31",
    // "image": "ghcr.io/missourimrdt/autonomy-jetpack:2025-05-10-00-39-31",
    // "build": {
        // "dockerfile": "Jammy.dockerfile"
        // "dockerfile": "JetPack.dockerfile"
    // },
```